### PR TITLE
sirius, q-e-sirius

### PR DESCRIPTION
**Pull request by simonpintarelli**
- sirius: fix bug `~/+shared` (`~shared` was not properly passed to cmake)
- sirius: make `+shared` the default
- q-e-sirius: explicitly depend on `sirius+shared`, fix gcc@10 workaround

### CODE REVIEW

File: var/spack/repos/builtin/packages/kokkos-nvcc-wrapper/package.py [hunk 1]
```diff
@@ -11,11 +11,11 @@ class KokkosNvccWrapper(CMakePackage):
...
     homepage = "https://github.com/kokkos/kokkos"
-    git = "https://github.com/kokkos/nvcc_wrapper.git"
+    git = "https://github.com/simonpintarelli/nvcc_wrapper.git"
 
-    version('3.1', url='https://github.com/kokkos/nvcc_wrapper/archive/3.1.tar.gz',
-            sha256="5171530763bb2845aa70651f3a8e5dc95b6eed76379323fde3205f97ed28ec8c",
-            default=True)
+    # version('3.1', url='https://github.com/kokkos/nvcc_wrapper/archive/3.1.tar.gz',
+    #         sha256="5171530763bb2845aa70651f3a8e5dc95b6eed76379323fde3205f97ed28ec8c",
+    #         default=True)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/kokkos-nvcc-wrapper/package.py]: Was this change intended to be part of this PR?

Comment (simonpintarelli) [reply var/spack/repos/builtin/packages/kokkos-nvcc-wrapper/package.py]: Oh, very sorry, this was not intended to slip into this PR. I updated kokkos_nvcc wrapper for my own. Should have used a feature branch ...

**Review by adamjstewart — APPROVED** [review]
